.pre-commit-config.yaml

```diff
@@ -101,152 +97,135 @@ repos:
   # Local checks for PEP headers and more
   - repo: local
     hooks:
-      - id: check-no-tabs
-        name: "Check tabs not used in PEPs"
-        language: pygrep
-        entry: '\t'
-        files: '^peps/pep-\d+\.rst$'
-        types: [text]
-
       - id: check-required-headers
         name: "PEPs must have all required headers"
-        language: pygrep
+        language: 'pygrep'
```

Comment:
Let's skip these unrelated changes, we're going to be removing these hooks soon anyway (some time after https://github.com/python/peps/pull/3275 is merged), and this is clearly a string.